src/pocapi.cpp

```diff
@@ -71,6 +87,9 @@ OVMS_Status* OVMS_ServerDelete(OVMS_Server* server) {
 OVMS_Status* OVMS_ServerStartFromConfigurationFile(OVMS_Server* server,
     OVMS_ServerGeneralOptions* general_options,
     OVMS_ServerMultiModelOptions* multi_model_specific_options) {
+    if (server == nullptr) {
```

Comment:
Here you add nullptr handling for 2 calls but leave all other calls missing of the check

Reply:
Added in later commit.